src/styles/blocks/stars.css

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+.stars--0 > *,
+.stars--1 :nth-child(n + 1),
+.stars--2 :nth-child(n + 2),
+.stars--3 :nth-child(n + 3),
+.stars--4 :nth-child(n + 4) {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
you can move these styles to .stars__star and then just redeclare  background-image depends on stars modifier